modules/hcp-ibmz-lpar-agents.adoc

```diff
@@ -33,77 +33,28 @@ random.trust_cpu=on rd.luks.options=discard
 <3> For installations on DASD-type disks, use `rd.dasd` to specify the DASD where {op-system-first} is to be installed. For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed.
 <4> Specify this parameter when you use an Open Systems Adapter (OSA) or HiperSockets.
 
-. Generate the `.ins` and `initrd.img.addrsize` files.
+. Download the the `.ins` and `initrd.img.addrsize` files from the `InfraEnv` resource.
 +
-The `.ins` file includes installation data and is on the FTP server. You can access the file from the HMC system. The `.ins` file contains details such as mapping of the location of installation data on the disk or FTP server, the memory locations where the data is to be copied.
+By default, the URL for the `.ins` and `initrd.img.addrsize` files is not available in the `InfraEnv` resource. You must edit the URL to fetch those artifacts.
 +
-[NOTE]
-====
-In {product-title} 4.16, the `.ins` file and `initrd.img.addrsize` are not automatically generated as part of boot-artifacts from the installation program. You must manually generate these files.
-====
-
-.. Run the following commands to get the size of the `kernel` and `initrd`:
+.. Update the kernel URL endpoint to include `ins-file`:
```

Comment:
In this update you've removed commands and in place have direct steps with no commands but examples. Would it be better to offer the user commands for how to update the kernel URL endpoint etc? It could be that I'm not familiar with this so my these may be obvious commands.